subscription-engine/kafka-ws/examples/With Dojot/WSClient/Client.js

```diff
@@ -0,0 +1,131 @@
+/* eslint-disable no-console */
+// Example WS client
+const fs = require('fs');
+const WebSocket = require('ws');
+const superagent = require('superagent');
+const { promisify } = require('util');
+
+const parseBoolean = (mode) => ((mode || false) && (mode.toString().toLowerCase().trim() === 'true' || Number(mode) > 0));
+
+// Parsing environment variables
+const dojotAddress = process.env.KAFKAWS_DOJOT_ADDRESS || 'dojot.iotplat';
```

Comment:
In the server certificate the SAN is as `kafka-ws-example`
I think it's cool to keep the same name here instead of `dojot.iotplat`

I believe that the two docker-compose (client and server) will know how to communicate, right?
Will the docker know how to create the internal DNS table for the correct containers?

If for this example, it is not necessary to change the `hosts` file, even better. Otherwise, there would have to be a README explaining the need to change the `hosts` file for the example to run successfully.